dimensiondata/src/test/java/org/jclouds/dimensiondata/cloudcontrol/features/ServerApiMockTest.java

```diff
@@ -56,7 +57,7 @@ public void testDeployServerReturnsUnexpectedError() throws InterruptedException
          serverApi().deployServer(ServerApiMockTest.class.getSimpleName(), "imageId", true, networkInfo,
                "administratorPassword");
          failBecauseExceptionWasNotThrown(ResourceNotFoundException.class);
-      } catch (ResourceNotFoundException e) {
+      } catch (HttpResponseException e) {
```

Comment:
Why does this need to be changed?

Reply:
On UNEXPECTED_ERROR, deployServer API will now throw HttpResponseException.
ResourceNotFound did not make any sense in that case.
HttpResponseException means that we don't know that is going on - error was unexpected.

Reply:
Then the assertion just before the `catch` needs to be modified, right?